extension/test/newman/collection.json

```diff
@@ -123,7 +160,132 @@
 					"response": []
 				},
 				{
-					"name": "Confirm product added",
+					"name": "Add config base product to list",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "75ebdfa1-4830-4477-b86b-1f4d001fedbc",
+								"exec": [
+									"pm.test('Status code is 200', function () {",
+									"    pm.response.to.have.status(200);",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"productId\" : \"402\"\n}"
+						},
+						"url": {
+							"raw": "{{domain}}{{endpoint_user_favorites_add}}",
+							"host": [
+								"{{domain}}{{endpoint_user_favorites_add}}"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Confirm simple product added",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "5cd67dd2-faf6-4720-8031-cbf3a65d0a32",
+								"exec": [
+									"pm.test('Status code is 200', function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"pm.test('Check that the config. is properly added', () => {",
+									"    const response = pm.response.json();",
+									"    pm.expect(response.products).to.be.a('array').to.not.be.empty;",
+									"    pm.expect(response.products[2].id).to.eq('554');",
```

Comment:
i would recommend checking previously added instead of relying on a position within a list `[2]`